x-pack/plugins/maps/public/layers/sources/es_source/es_source.js

```diff
@@ -294,6 +304,7 @@ export class AbstractESSource extends AbstractVectorSource {
     }, {});
 
     const indexPattern = await this.getIndexPattern();
+    const { SearchSource } = getSearchService();
```

Comment:
What is the difference between `getSearchService` and `getSearchSourceType` and why does this usage of `new SearchSource` use `getSearchService` while the other uses use `getSearchSourceType`?

Reply:
Thank you, it was my issue. Fixed.